src/main/resources/application.yml

```diff
@@ -1,6 +1,10 @@
 logging:
     level:
-        be.xplore.githubmetrics: ${LOGGING_LEVEL:INFO}
+        be.xplore.githubmetrics:
+            ROOT: INFO
+            githubadapter.config.ratelimiting: TRACE
+            githubadapter.ratelimiting: TRACE
+
```

Comment:
should this be committed? this was for your testing I assume?

Reply:
done